controllers/configmap_controller.go

```diff
@@ -123,7 +123,7 @@ func (r *ConfigMapReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 }
 
 // parseHosts gets the lists of hosts specified in the configmap's data
-func (r *ConfigMapReconciler) parseHosts(configMapData map[string]string) ([]*instances.InstanceInfo, error) {
+func parseHosts(configMapData map[string]string) ([]*instances.InstanceInfo, error) {
```

Comment:
Given you are changing this, please rename the function to `parseInstances` and mention it in the commit.